modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,138 @@
+
+* Before this update, in some instance when applications are deleted, their application environment card would remain in a half-deleted state until the page was refreshed. With this update, the application environment card is removed from the UI automatically after the Argo CD application is deleted. link:https://issues.redhat.com/browse/GITOPS-2677[GITOPS-2677]
+
+* Before this update, when a user deleted apps and namespaces in Argo CD, in some cases, the {gitops-title} plugin would display an error message. This update fixes the issue by updating some components from the dynamic plugin SDK and provides better error handling. As a result, the error message is not displayed after deleting apps or namespaces. link:https://issues.redhat.com/browse/GITOPS-2746[GITOPS-2746]
+
+* Before this update, the password for the `admin` role was used to reset to the default password when it was updated by the Argo CD CLI or the Argo CD UI. This update fixes the issue and the user can now update the admin password and the password is no longer set to default. link:https://issues.redhat.com/browse/GITOPS-3581[GITOPS-3581]
+
+[id="known-issues-1-12-0_{context}"]
+== Known Issues
+
+* There is currently a known issue that changes the functionality of routing in the OpenShift Console Dynamic plugin, `dynamic-console-sdk`, that is used by the {gitops-title} Dynamic Plugin. This issue causes the horizontal navigation bar, which is used to switch between the *Application Overview* page and the *Deployment History* page for an application, on the *Application Overview* page to not function correctly. link:https://issues.redhat.com/browse/GITOPS-4232[GITOPS-4232]
++
+Workaround: To view the **Deployment History** page of an application, use the *Deployment History* link on the application instead of the link on the horizontal navigation bar on the *Application Overview* page. This issue has been observed on all supported cluster versions that use the {gitops-title} Dynamic Plugin.
+
+* There is currently a known issue on the {OCP} 4.15 cluster for the console {gitops-title} Dynamic Plugin where users can experience some delay when fetching application data. link:https://issues.redhat.com/browse/GITOPS-4234[GITOPS-4234]
++
+Workaround: No workaround currently exists for this issue, so you must wait for the fetching of the application data to complete.
+
+* There is currently a known issue that causes the {gitops-title} Dynamic Plugin to be disabled on the {OCP} 4.15 cluster. link:https://issues.redhat.com/browse/GITOPS-4231[GITOPS-4231]
++
+Workaround: Perform the following steps:
++
+. Install the {gitops-title} Operator in your cluster
+. In the *Administrator* perspective of the web console, navigate to *Home* -> *Overview*.
+. On the *Overview* tab, click the *Dynamic plugins* link in the *Status* section. 
+. To enable the {gitops-title} Dynamic Plugin, click *gitops-plugin* and then click *Enabled*. 
+
```

Comment:
Add a plus before to indent the sentence:

Some time, a notification with the message “Web console update is available” is displ

Reply:
Fixed.